src/scenes/home/profile/profile.css

```diff
@@ -10,3 +10,8 @@
   width: 220px;
   margin-top: 50px;
 }
+
+.discourseImage:hover {
+  opacity: 0.5;
+  filter: alpha(opacity=50); /* For IE8 and earlier */
+}
```

Comment:
This CSS class name should be renamed if its styles are also being used for the Scholarships image.  It's now a multi-use CSS class, applied to more images than the pre-existing Discourse image.

Reply:
Will do